css/bourbon/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:2.2.2
+MAINTAINER Joe Ferris <jferris@thoughtbot.com>
+RUN mkdir -p /opt/include
+RUN mkdir /src
+RUN mkdir /build
+RUN gem install sass
+RUN gem install bourbon -v 4.2.3
```

Comment:
personally no ... but i'm an iOS dev. I'm not sure how the web does things. I guess if you're not using bourbon then having it installed wouldn't be hurting you.